build.sh

```diff
@@ -93,8 +93,11 @@ cd ../..
 
 echo ""
 echo "Copying extra plugins..."
-cp -r plugins/* ckeditor/plugins/
-
+if [ "$(uname)" == "Darwin" ]; then
+	cp -R plugins/ ckeditor/plugins/
```

Comment:
Another thing, on original version there were `cp -r plugins/* ckeditor/plugins/` for OSX there is `-cp -R plugins/ ckeditor/plugins/` (no star) which I think works differently (at leas on Linux).
Example:
```$ tree
├── dest
└── source
    ├── s1
    │   └── file_1.txt
    └── s2
        └── file_2.txt
```

With star

```
$ cp -R source/* dest/
$ tree
├── dest
│   ├── s1
│   │   └── file_1.txt
│   └── s2
│       └── file_2.txt
└── source
    ├── s1
    │   └── file_1.txt
    └── s2
        └── file_2.txt
```

Without star

```
$ cp -R source/ dest/
$ tree
├── dest
│   └── source
│       ├── s1
│       │   └── file_1.txt
│       └── s2
│           └── file_2.txt
└── source
    ├── s1
    │   └── file_1.txt
    └── s2
        └── file_2.txt
```